styles/index.css

```diff
@@ -0,0 +1,26 @@
+header {text-align: center;}
```

Comment:
You can change how this is formatted property and value on its own line (like what you did on lines 3-5) to be in line with the rest of the formatting in this file